CMakeLists.txt

```diff
@@ -473,3 +473,44 @@ ExternalProject_Add(aws-sdk-cpp
     -DCMAKE_INSTALL_PREFIX:PATH=${TRITON_THIRD_PARTY_INSTALL_PREFIX}/aws-sdk-cpp
   PATCH_COMMAND python3 ${CMAKE_CURRENT_SOURCE_DIR}/tools/install_src.py --src <SOURCE_DIR> ${INSTALL_SRC_DEST_ARG}
 )
+
+#
+# Build OpenTelemetry C++ library
+# [FIXME] Remove WIN flag when windows build works on Triton
+#         JIRA DLIS-4786
```

Comment:
Thanks for adding it. This still feels a bit jumbled... is the JIRA ticket splitting up the sentence? Is the first or second part related to the JIRA ticket? We need to get a bit more consistent with how we leave these comments, but these two show more common ways that could serve as good examples.

- https://github.com/triton-inference-server/server/blob/11008c548d6e79247bac0f9a43afd33e90df541a/Dockerfile.sdk#L256
- https://github.com/triton-inference-server/server/blob/ebd7aab1e929a8f3d871bd9ec6b759b9a4db7bb9/qa/common/test_util.py#L99